applications/tweets/helpers.py

```diff
@@ -11,6 +11,23 @@
 
 logger = logging.getLogger(__name__)
 
+def create_address(processed_address=None,full_address=None,tweet=None):
+    city = processed_address.get("city")
+    allowed_cities = ["Gaziantep", "Malatya", "Batman", "Bingöl", "Elazığ", "Kilis", "Diyarbakır", "Mardin", "Siirt", "Şırnak", "Van", "Muş", "Bitlis", "Hakkari", "Adana", "Osmaniye","Hatay"]
```

Comment:
Bunu yapmak yerine aşağıdaki regexi kullanarak uppercase ve typoları da yakalayan bir kurgu yapalım
```suggestion
(?i)((gaz[ıiİI]antep)|(malatya)|(batman)|(b[ıiIİ]ng[oöOÖ]l)|(elaz[Iİıi][gğ])|(kilis)|(diyarbak[ıiIİ]r)|(mardin)|(siirt)|([SsŞş][ıiIİ]rnak)|(van)|(mu[sşSŞ])|(bitlis)|(hakkari)|(adana)|(osmaniye)|(hatay)|(kahramanmara[sşSŞ])|(mara[SŞsş])|(antep))

```